fix: improve support for SVG allow crop skipping

preda-bogdan opened the pull request:
### Summary
I've improved support for SVG, it should allow use of SVG inside the Logo and will allow crop skipping since it is not possible on SVG.

### Will affect visual aspect of the product
NO

### Test instructions
1. Upload a SVG using [SVG Support](https://wordpress.org/plugins/svg-support/) plugin.
2. Set the SVG as a LOGO, check that you are allowed to skip crop.
3. Check that all settings are applied correctly, eg. Size of Logo.

Closes #3350.

cristian-ungureanu commented on assets/apps/components/src/Common/MediaLibrary.js:
```diff
@@ -14,9 +14,12 @@ const getAttachmentsCollection = (ids) => {
 };
 
 const mustBeCropped = (flexW, flexH, dstW, dstH, imgW, imgH) => {
```
@preda-bogdan seems like this function returns false no matter what.

cristian-ungureanu commented on assets/apps/components/src/Common/MediaLibrary.js:
```diff
@@ -46,7 +49,7 @@ const mustBeCropped = (flexW, flexH, dstW, dstH, imgW, imgH) => {
+	return false;
 };
 
 const calculateImageSelectOptions = (attachment, controller) => {
```
Please add some comments here to explain the mechanism behind this.

cristian-ungureanu commented on assets/apps/components/src/Common/MediaLibrary.js:
```diff
@@ -61,7 +64,7 @@ const calculateImageSelectOptions = (attachment, controller) => {
 	let yInit = parseInt(currentCropControl.params.height, 10);
 
-	const ratio = xInit / yInit;
+	const ratio = 4 / 1;
```
This seems weird. Wouldn't the ratio be always 4:1?

preda-bogdan commented:
@cristian-ungureanu I will do some cleanup here, as they were left from my testing. Are there any other changes that would be required here before proceeding with testing?

cristian-ungureanu commented:
No, everything else looks good 👍

preda-bogdan commented:
@irinelenache I've updated the build and fixed the aspect ratio issue. I've updated your tastewp instace to test.
The issues should be solve now, you can check it out on the instance you provided and also on a new instance.

Let me know how it goes.
Thank you!

irinelenache commented:
@preda-bogdan Tested again and now everything works fine, thank you 🚀
